src/shared/models/fiat/fiat.service.ts

```diff
@@ -1,9 +1,19 @@
 import { Injectable } from '@nestjs/common';
+import { Config } from 'src/config/config';
 import { FiatRepository } from 'src/shared/models/fiat/fiat.repository';
 import { Fiat } from './fiat.entity';
 
 @Injectable()
 export class FiatService {
+  private ipCountryToCurrency: { [key: string]: string } = {
+    DE: 'EUR',
+    AT: 'EUR',
+    CH: 'CHF',
+    LI: 'CHF',
+    IT: 'EUR',
+    FR: 'EUR',
+  };
```

Comment:
Maybe it makes sense to have a `ipCountryDefaultList` which then can be used for both `ipCountryToCurrency` and `ipCountryToLanguage` 🤔  With that we could define the key with the new `IpCountryDefaultList` 
@davidleomay wdym?

Reply:
Maybe also something like this in `geo-location.service.ts`? 🤔

```ts
  async getDefaultLanguage(ip: string): Promise<string> {
    const geoLocation = GeoIp.lookup(ip);
    const ipCountry = geoLocation?.country;
    switch (ipCountry) {
      case 'DE':
      case 'AT':
      case 'CH':
      case 'LI':
        return 'DE';
      case 'IT':
        return 'IT';
      case 'FR':
        return 'FR';
      default:
        return 'EN';
    }
  }

  async getDefaultCurrency(ip: string): Promise<string> {
    const geoLocation = GeoIp.lookup(ip);
    const ipCountry = geoLocation?.country;
    const isEU = geoLocation?.eu === '1';
    switch (ipCountry) {
      case 'CH':
      case 'LI':
        return 'CHF';
      default:
        return isEU ? 'EUR' : 'USD';
    }
  }
  ```

Reply:
Actually it doesn't look like the functions in `geo-location.service.ts` need to be `async`?